docs/orchestration/kubernetes-yaml/README.md

```diff
@@ -134,7 +137,7 @@ kubectl create -f https://github.com/minio/minio/blob/master/docs/orchestration/
 deployment "minio-deployment" created
 ```
 
-### Step 3: Create Minio Service
+### Create Minio Service
 
 Now that you have a Minio deployment running, you may either want to access it internally (within the cluster) or expose it as a Service onto an external (outside of your cluster, maybe public internet) IP address, depending on your use case. You can achieve this using Services. There are 3 major service types — default type is ClusterIP, which exposes a service to connection from inside the cluster. NodePort and LoadBalancer are two types that expose services to external traffic.
```

Comment:
We're referring to Kubernetes Service here (https://kubernetes.io/docs/concepts/services-networking/service/) not a service in general, hence the captialization. We can change it but I felt this is a Kubernetes feature so it made sense to write it as Service.